Use `TranslationModelForm` to implement `ActiveLanguageMixin` (#80)
zostera/django-modeltrans: docs/pages/admin.rst

.. _admin:

Admin
=====

By default, each field is displayed for each language configured for django-modeltrans.
This might work for a couple of languages, but with 2 translated fields and 10 languages,
it already is a bit unwieldy.

> A mixin is provided to show only the default language (`settings.LANGUAGE_CODE`) and

Reviewer: ```suggestion
The `ActiveLanguageMixin` is provided to show only the default language (`settings.LANGUAGE_CODE`) and
```
more direct

    @admin.register(Blog)
    class BlogAdmin(ActiveLanguageMixin, admin.ModelAdmin):
        pass

> This uses `modeltrans.forms.TranslationModelForm`, if you need more customization,

Reviewer: ```suggestion
`ActiveLanguageMixin` uses `modeltrans.forms.TranslationModelForm`, if you need more customization,
```

> it can be used directly::

Reviewer: ```suggestion
the latter can be used directly::
```
